### PR TITLE
[SRVLS] [SRVCOM-1233] Updating 4.6 to new guidelines

### PR DESCRIPTION
**Pull request by abrennan89**

### CODE REVIEW

File: modules/serverless-jaeger-config.adoc [hunk 1]
```diff
@@ -62,22 +52,17 @@ spec:
 
 Access the Jaeger web console to see tracing data. You can access the Jaeger web console by using the `jaeger` route.
 
-. Get the `jaeger` route's hostname by entering the following command:
+. Get the hostname of the `jaeger` route:
```
Comment (adellape) [inline modules/serverless-jaeger-config.adoc:55]: s/hostname/host name/

[Per](https://redhat-documentation.github.io/supplementary-style-guide/#host-name)

File: modules/serverless-metering-queries.adoc [hunk 2]
```diff
@@ -3,13 +3,10 @@
 
 [id="queries-metering-serverless_{context}"]
 = Queries for Knative Serving metering
-The following `ReportQuery` resources reference the example `DataSources` provided.
 
-[id="knative-service-cpu-usage-query_{context}"]
-== Query for CPU usage in Knative Serving
-
-.YAML file
+The following `ReportQuery` resources reference the example `ReportDataSource` resources provided.
```
Comment (adellape) [inline modules/serverless-metering-queries.adoc:7]: s/./:/

Since it mentions "the following".

File: modules/serverless-rn-1-11-0.adoc [hunk 3]
```diff
@@ -11,16 +11,13 @@
...
-* {ServerlessProductName} Functions is now delivered as a Developer Preview through the standard Knative `kn` CLI installation. This feature is not yet supported by Red Hat for production deployments, but can be used for development and testing. For more information about using {ServerlessProductName} Functions through the `kn func` CLI, see the link:https://openshift-knative.github.io/docs/docs/functions/about-functions.html[{ServerlessProductName} Functions Developer Preview documentation].
+* {ServerlessProductName} Functions is now delivered as a Developer Preview through the standard Knative `kn` CLI installation. This feature is not yet supported by Red Hat for production deployments, but can be used for development and testing. For more information about using {ServerlessProductName} Functions through the `kn func` CLI plugin, see the link:https://openshift-knative.github.io/docs/docs/functions/about-functions.html[{ServerlessProductName} Functions Developer Preview documentation].
```
Comment (adellape) [inline modules/serverless-rn-1-11-0.adoc:14]: s/plugin/plug-in/

File: modules/serverless-service-mesh-resources.adoc [hunk 4]
```diff
@@ -122,14 +111,12 @@ spec:
     kind: Service
     name: istio-ingressgateway
 ----
-
 <1> The {product-title} route must be created in the same namespace as the
 ServiceMeshControlPlane. In this example, the ServiceMeshControlPlane is
```
Comment (adellape) [inline modules/serverless-service-mesh-resources.adoc:115]: Should these `ServiceMeshControlPlane` instances be marked up?

**Review by adellape — COMMENTED** [review]
Very nice! Only a few suggestions.

File: modules/serverless-service-mesh-resources.adoc (hunk 4 above)
Comment (abrennan89) [reply modules/serverless-service-mesh-resources.adoc:115]: Good call, checked in the OSSM docs and they use "`ServiceMeshControlPlane` resource" now so I changed it to that.

**Pull request merged**